.github/workflows/pylint.yml

```diff
@@ -21,6 +21,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install pylint
+        pip install .
+        pip install -r requirements-dev.txt
+        pip install -r requirements-testing.txt
```

Comment:
The pylint test checks all imports in the entire Python code. Therefore, we need to install all of them before running pylint.